app/config.dev.json

```diff
@@ -9,5 +9,8 @@
   },
   "email": {
     "endpoint": "http://localhost:3000/api/emails"
+  },
+  "REACT_APP_GPIB_URL": {
+    "endpoint": "https://testapi.getpaidinbitcoin.com.au"
```

Comment:
Lets get this from https://github.com/dltxio/idem/blob/master/assets/sites.json#L7 and use key `1x01`